### PR TITLE
Added VIA keymap to YMDK bface keyboard

### PR DESCRIPTION
**Pull request by Desp-Key**
## Description

Added keymap files to make YMDK bface VIA compatible.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ymdk/bface/keymaps/via/rules.mk [hunk 1]
```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = no
```
Comment (zvecr) [inline keyboards/ymdk/bface/keymaps/via/rules.mk:4]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]
https://github.com/qmk/qmk_firmware/blob/master/keyboards/ymdk/bface/config.h#L20-L21 is also not unique and will need updating.

File: keyboards/ymdk/bface/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H 
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    //Base Layer
+    [0] = LAYOUT_60_ansi(
```
Comment (fauxpark) [inline keyboards/ymdk/bface/keymaps/via/keymap.c:22]: VIA keymaps should use `LAYOUT_all` where possible.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ymdk/bface/keymaps/via/rules.mk (hunk 1 above)
Comment (Desp-Key) [reply keyboards/ymdk/bface/keymaps/via/rules.mk:4]: When I don't include those two I get the following error message: `error "Enabling Mousekeys/Extrakeys and Raw HID at the same time is not currently supported on V-USB."`
With them my keymap seems to work in VIA.

Comment (fauxpark) [reply keyboards/ymdk/bface/keymaps/via/rules.mk:4]: I'm not sure how that's possible given your fork appears to be up to date - enabling mouse/extra with VIA is in fact now supported - and this PR has been opened from the master branch (which, as a side note, is [not really recommended](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)).

Comment (Desp-Key) [reply keyboards/ymdk/bface/keymaps/via/rules.mk:4]: I updated  my fork and indeed it now works. Thanks for the advice!

Comment (Erovia) [reply keyboards/ymdk/bface/keymaps/via/rules.mk:4]: So could you remove them from the rules.mk?

Comment (Desp-Key) [reply keyboards/ymdk/bface/keymaps/via/rules.mk]: Oh oops! Done.

**Review by Erovia — APPROVED** [review]

**Review by Erovia — APPROVED** [review]

File: keyboards/ymdk/bface/config.h [hunk 3]
```diff
@@ -17,8 +17,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
-#define VENDOR_ID       0x20A0
-#define PRODUCT_ID      0x422D
+#define VENDOR_ID       0x594D
```
Comment (fauxpark) [inline keyboards/ymdk/bface/config.h:20]:
```suggestion
#define VENDOR_ID       0x594D // "YM"
```

File: keyboards/ymdk/bface/config.h [hunk 4]
```diff
@@ -17,8 +17,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
-#define VENDOR_ID       0x20A0
-#define PRODUCT_ID      0x422D
+#define VENDOR_ID       0x594D 
+#define PRODUCT_ID      0x4266
```
Comment (fauxpark) [inline keyboards/ymdk/bface/config.h:21]:
```suggestion
#define PRODUCT_ID      0x4266 // "Bf"
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**